source/CMakeLists.txt

```diff
@@ -85,14 +85,6 @@ if (YOTTA_CFG_MICROBIT_CONFIGFILE)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${YOTTA_FORCE_INCLUDE_FLAG} \"${YOTTA_CFG_MICROBIT_CONFIGFILE}\"")
 endif ()
 
-if(CMAKE_COMPILER_IS_GNUCC)
-  file(REMOVE "asm/CortexContextSwitch.s")
-  configure_file("asm/CortexContextSwitch.s.gcc" "asm/CortexContextSwitch.s" COPYONLY)
-else()
-  file(REMOVE "asm/CortexContextSwitch.s")
-  configure_file("asm/CortexContextSwitch.s.armcc" "asm/CortexContextSwitch.s" COPYONLY)
-endif()
-
```

Comment:
Doesn't this need to be updated to move the `asm/TOOLCHAIN_GCC_ARM/CortexContextSwitch.s` or `asm/TOOLCHAIN_ARM_STD/CortexContextSwitch.S` into `asm/CortexContextSwitch.s`?

Reply:
Yotta appears to pay attention to TOOLCHAIN_* directories, as there are plenty in mbed-classic which cause no conflicts, though I do need to modify the line below to point at the new location.

Reply:
So in summary, yes.